### DISCUSSION TITLE
“说说”页面模板的标题沿用的是主页标题，如何修改？

### DISCUSSION THREAD

**Discussion by ERROR403XD** [solstice23/argon-theme#596] [Q&A]
我刚开始碰wp，html和php，折腾了几天，发现留言板和时间轴这两个页面模板的title都是固定的，说说没有自己的title，而且这几个页面都无法通过后台添加title元数据来修改标题
研究了一下只发现定义了title字段的head标签都是通过get_header完成的，但我在header.php里没找到给这几个页面模板赋予title字段的代码，在function.php里也没有找到钩子在做这个事情
现在我就很好奇，是什么机制导致了说说和其他两个页面模板在这个问题上的区别呢？它们的head标签都是通过get_header完成的，为什么能对两个页面分别提供不同的title字段，而对另一个没有提供呢？
希望来大哥教教，这两天都在用GPT老师学着改php文件，但这个问题G老师好像回答不出来……

**Comment by crowya** [comment]
我比你还小白，我是用JavaScript来替换“说说”标题的...

**Comment by Arilt** [comment]
去修改shuoshuo.hp以及模板目录里的template-shuoshuo.php试试
